pkg/kubelet/config/file_linux.go

```diff
@@ -89,7 +89,7 @@ func (s *sourceFile) processEvent(e *inotify.Event) error {
 	var eventType podEventType
 	switch {
 	case (e.Mask & inotify.IN_ISDIR) > 0:
-		glog.V(1).Infof("Not recursing into config path %q", s.path)
+		glog.V(1).Infof("Not recursing into manifest path %q", s.path)
```

Comment:
Same info vs. error comment as above. If this message happens regularly in normal operation, then how useful is it? Should we raise the log level?